### PR TITLE
Reject CVE-2021-46388

### PR DESCRIPTION
**Pull request by CERT-VDE**

### CODE REVIEW

File: 2021/46xxx/CVE-2021-46388.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "CVE_data_meta": {
```
Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'WAGO 750-8212 PFC200 G2 2ETH RS', 'version': {'version_data': [{'version_value': 'Frimware version: 03.05.10(17)'}]}}]}, 'vendor_name': 'Wago'}]}}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'Privilege Escalation'}]}]}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {'reference_data': [{'name': 'http://rejected', 'refsource': 'CONFIRM', 'url': 'http://rejected'}]}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/46xxx/CVE-2021-46388.json [hunk 2]
```diff
@@ -2,30 +2,10 @@
...
```
Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {'not': {}}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {'not': {}}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {'not': {}}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/46xxx/CVE-2021-46388.json [hunk 3]
```diff
@@ -2,30 +2,9 @@
...
```
Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {}

Comment (cve-team) [inline 2021/46xxx/CVE-2021-46388.json:2]: {} is not allowed for {}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by CERT-VDE** [comment]
Hello Team,

what's missing here? After we had no success with the guide of the onboarding slides we used the JSON scheme for a reject entry, also with validation errors. Finally we used an existing reject as template and it seems the validation errors are gone.
We requested the review four days ago, are we missing something?

Thank you,

CERT@VDE/jochen

**Pull request closed without merging**